bin/rbenv-each

```diff
@@ -46,7 +71,13 @@ failed_rubies=""
 
 trap "exit 1" INT
 
-for ruby in $(rbenv versions --bare); do
+if [ -n "$aliases" ]; then
+    opts=
+else
+    opts="--skip-aliases"
```

Comment:
This is a feature only available in edge version of rbenv. We could only potentially ship this PR after rbenv 1.0 gets released. Also, we should ensure that `rbenv each` keeps working even with rbenv 0.4 for people who didn't upgrade.